gnovm/stdlibs/std/banker.gno

```diff
@@ -66,7 +68,17 @@ func GetBanker(bt BankerType) Banker {
 	if bt >= maxBanker {
 		panic("invalid banker type")
 	}
-	return banker{bt}
+
+	var pkgAddr string
+	if bt == BankerTypeOrigSend {
+		pkgAddr = GetOrigPkgAddr().String()
```

Comment:
Can we have an assertion here, checking that `GetOrigPkgAddr() == CurrentRealm().Addr()`?

This makes it so that any other called realms cannot use the OrigSend.

Reply:
c2f64362a3a4f6dc6fd1a1295eba83b501d2d6ca